core/unit_test/TestDefaultDeviceTypeInit.hpp

```diff
@@ -357,6 +357,8 @@ TEST(defaultdevicetypeinit, commandline_args_nthreads_numa) {
 }
 #endif
 
+// FIXME_SYCL CI doesn't have a GPU yet
+#ifndef KOKKOS_ENABLE_SYCL
```

Comment:
does not feel like the right macro if this about Jenkins

Reply:
We always use the default device for now so we can enable these tests again.